Formula/b/ballerina.rb

```diff
@@ -6,8 +6,8 @@ class Ballerina < Formula
   license "Apache-2.0"
 
   livecheck do
-    url "https://github.com/ballerina-platform/ballerina-lang.git"
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
+    url "https://ballerina.io/downloads/"
```

Comment:
we should check against the download page rather than the git (as there is some delay with the release), but I could not get this livecheck update working for some reason. cc @samford